### ISSUE TITLE
Solidity 0.6.* - Error: invalid fragment object

### ISSUE THREAD

**Issue by ItsNickBarry** [cgewecke/buidler-gas-reporter#13]
I am encountering an issue which appears to be identical to that described in https://github.com/cgewecke/eth-gas-reporter/issues/191.  I don't know whether  its recurrence is specific to `buidler-gas-reporter`.

The problem appears to be with this syntax:

```solidity
receive () external payable {}
```

I've made sure that the `eth-gas-reporter` dependency is up-to-date:

```
$ cat yarn.lock | grep eth-gas-reporter
    eth-gas-reporter "^0.2.13"
eth-gas-reporter@^0.2.13:
  resolved "https://registry.yarnpkg.com/eth-gas-reporter/-/eth-gas-reporter-0.2.15.tgz#663752ac05bad1054c3c54d6fa611865a6a75fc1"
```

By the way, casting an `address` to `payable` does not seem to raise any issues.

**Comment by cgewecke** [comment]
@ItsNickBarry 

Very sorry for the crazy delay responding here, must have missed this.... I think this got fixed in eth-gas-reporter #206. So if EGR resolves to 0.2.17 it should work.

Please just lmk if not.

[1]: https://github.com/cgewecke/eth-gas-reporter/pull/206

**Comment by cgewecke** [comment]
(Apparently when you create a repo you're not automatically "watching" it? I'm now a watcher fwiw.)
